ApplyToBecomeInternal/ApplyToBecomeInternal/Controllers/DashboardController.cs

```diff
@@ -17,7 +17,10 @@ public DashboardController(IProjects projects)
 		}
 		public IActionResult Index()
 		{
-			return View();
+			var ongoingProjects = _projects.GetAllProjects();
+			ViewBag.TotalOngoingProjects = ongoingProjects.Count();
```

Comment:
We need a view model for this. We should avoid using the `ViewBag` where possible.